client/basic-authorship/src/lib.rs

```diff
@@ -71,4 +71,4 @@
 
 mod basic_authorship;
 
-pub use crate::basic_authorship::{ProposerFactory, Proposer};
+pub use crate::basic_authorship::{ProposerFactory, Proposer, DEFAULT_MAX_BLOCK_SIZE};
```

Comment:
here as DEFAULT_MAX_BLOCK_SIZE was public in the module but not reexported I guess it was a miss.